<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EntityWidget</class>
 <widget class="QWidget" name="EntityWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>280</width>
    <height>823</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Children:</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QTreeView" name="childrenTree">
          <attribute name="headerVisible">
           <bool>false</bool>
          </attribute>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QSplitter" name="splitter">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="childrenCollapsible">
        <bool>false</bool>
       </property>
       <widget class="QFrame" name="SelectedItemFrame">
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
        <property name="lineWidth">
         <number>2</number>
        </property>
        <property name="midLineWidth">
         <number>1</number>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <property name="leftMargin">
          <number>2</number>
         </property>
         <property name="topMargin">
          <number>2</number>
         </property>
         <property name="rightMargin">
          <number>2</number>
         </property>
         <property name="bottomMargin">
          <number>2</number>
         </property>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <spacer name="horizontalSpacer_4">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLabel" name="lblSelectedItemIcon">
             <property name="text">
              <string/>
             </property>
             <property name="pixmap">
              <pixmap resource="../res/HyEditor.qrc">:/icons16x16/Harmony.png</pixmap>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="lblSelectedItemText">
             <property name="font">
              <font>
               <weight>75</weight>
               <bold>true</bold>
               <underline>true</underline>
              </font>
             </property>
             <property name="text">
              <string>TextLabel</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_5">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <widget class="PropertiesTreeView" name="propertyTree">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="alternatingRowColors">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="ChildrenToolbar">
       <item>
        <widget class="QToolButton" name="btnAddChild">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="btnAddChildPrimitive">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="btnInsertBoundingVolume">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="btnInsertPhysics">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="btnAddScissor">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
  <action name="actionRenameState">
   <property name="icon">
    <iconset resource="../res/HyEditor.qrc">
     <normaloff>:/icons16x16/generic-rename.png</normaloff>:/icons16x16/generic-rename.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Rename State</string>
   </property>
   <property name="toolTip">
    <string>Rename state</string>
   </property>
  </action>
  <action name="actionAddState">
   <property name="icon">
    <iconset resource="../res/HyEditor.qrc">
     <normaloff>:/icons16x16/generic-add.png</normaloff>:/icons16x16/generic-add.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Add State</string>
   </property>
   <property name="toolTip">
    <string>Add a new state for this entity</string>
   </property>
  </action>
  <action name="actionRemoveState">
   <property name="icon">
    <iconset resource="../res/HyEditor.qrc">
     <normaloff>:/icons16x16/edit-delete.png</normaloff>:/icons16x16/edit-delete.png</iconset>
   </property>
   <property name="text">
    <string>Remo&amp;ve State</string>
   </property>
   <property name="toolTip">
    <string>Remove the currently selected state</string>
   </property>
  </action>
  <action name="actionOrderStateBackwards">
   <property name="icon">
    <iconset resource="../res/HyEditor.qrc">
     <normaloff>:/icons16x16/generic-left.png</normaloff>:/icons16x16/generic-left.png</iconset>
   </property>
   <property name="text">
    <string>Order State &amp;Backwards</string>
   </property>
   <property name="toolTip">
    <string>Shift the index of the current state back by 1</string>
   </property>
  </action>
  <action name="actionOrderStateForwards">
   <property name="icon">
    <iconset resource="../res/HyEditor.qrc">
     <normaloff>:/icons16x16/generic-right.png</normaloff>:/icons16x16/generic-right.png</iconset>
   </property>
   <property name="text">
    <string>Order State For&amp;wards</string>
   </property>
   <property name="toolTip">
    <string>Shift the index of the current state forward by 1</string>
   </property>
  </action>
  <action name="actionAddPrimitive">
   <property name="icon">
    <iconset resource="../res/HyEditor.qrc">
     <normaloff>:/icons16x16/items/Primitive-New.png</normaloff>:/icons16x16/items/Primitive-New.png</iconset>
   </property>
   <property name="text">
    <string>Add a Primitive shape as a child</string>
   </property>
   <property name="toolTip">
    <string>Inserts a primitive shape as a child, at the selected node</string>
   </property>
  </action>
  <action name="actionInsertBoundingVolume">
   <property name="icon">
    <iconset resource="../res/HyEditor.qrc">
     <normaloff>:/icons16x16/items/BoundingVolume-New.png</normaloff>:/icons16x16/items/BoundingVolume-New.png</iconset>
   </property>
   <property name="text">
    <string>Insert Bounding Volume</string>
   </property>
   <property name="toolTip">
    <string>Inserts a bounding volume at the selected node to detect collision, mouse input, hitbox, etc.</string>
   </property>
  </action>
  <action name="actionInsertPhysicsBody">
   <property name="icon">
    <iconset resource="../res/HyEditor.qrc">
     <normaloff>:/icons16x16/items/Physics-New.png</normaloff>:/icons16x16/items/Physics-New.png</iconset>
   </property>
   <property name="text">
    <string>Insert Physics Body</string>
   </property>
   <property name="toolTip">
    <string>Inserts a physics body at the selected node that simulates forces and velocity</string>
   </property>
  </action>
  <action name="actionAddScissorBox">
   <property name="text">
    <string>Add Scissor Box</string>
   </property>
   <property name="toolTip">
    <string>Inserts a scissor box which culls away a portion of the visual</string>
   </property>
  </action>
  <action name="actionAddSelectedChild">
   <property name="icon">
    <iconset resource="../res/HyEditor.qrc">
     <normaloff>:/icons16x16/generic-add.png</normaloff>:/icons16x16/generic-add.png</iconset>
   </property>
   <property name="text">
    <string>Add Selected Child</string>
   </property>
   <property name="toolTip">
    <string>Add the selected item in Project Explorer as a child</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PropertiesTreeView</class>
   <extends>QTreeView</extends>
   <header>PropertiesTreeView.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../res/HyEditor.qrc"/>
 </resources>
 <connections/>
</ui>
